barretenberg/acir_tests/flows/prove_tube.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -eu
+
+VFLAG=${VERBOSE:+-v}
+BFLAG="-b ./target/program.json"
+FLAGS="-c $CRS_PATH $VFLAG"
+
+$BIN client_ivc_prove_output_all $VFLAG -c $CRS_PATH -b ./target/program.json
```

Comment:
This line should not be here, right? We don't just want testing flows--the client produces their proof, and prove tube should be the first function that a rollup node runs on their input.